.github/workflows/deploy-landing-page.yaml

```diff
@@ -36,6 +36,8 @@ concurrency:
 jobs:
   # Build job
   build:
+    env:
+      NEXT_PUBLIC_ANALYTICS_WRITE_KEY: semVJB1hXAeudvf1VExo0gdJcJQheGBZ
```

Comment:
This should be in an environment variable in GitHub and reused.

Reply:
Same thing as above.